src/cryptography/hazmat/primitives/asymmetric/rsa.py

```diff
@@ -120,8 +120,11 @@ def generate_private_key(public_exponent, key_size, backend):
 
 
 def _verify_rsa_parameters(public_exponent, key_size):
-    if public_exponent < 3:
-        raise ValueError("public_exponent must be >= 3.")
+    if public_exponent != 3 and public_exponent < 65537:
```

Comment:
Can this just be `public_expontent not in [3, 65537]`? Do we care about other values?

Reply:
We can constrain all the way to just those two. I was matching OpenSSL 3.0 but I'm fine with being more strict.

Reply:
I'd be hesitant to restrict this more than OpenSSL does.

Reply:
We still parse any existing key, we just won't generate keys with weird values. I'm unaware of any production utility for large public exponents except for stunt public key tricks.

Reply:
If it's only an issue when generating new keys from within PyCA, that's less of a concern, but I would still worry about someone porting OpenSSL-based code from another language or trying to build something using PyCA which interoperates with another OpenSSL-based application. If that application happened to require another exponent, you're making things hard for the developer using PyCA, and I'm not sure this stricter check is adding much value.